.github/workflows/image-build.yaml

```diff
@@ -64,19 +129,21 @@ jobs:
               # set latest tag for default branch
               type=raw,value=latest,enable={{is_default_branch}}
 
-    name: Build Image (${{ matrix.image.image }})
+    name: Build Bundle Image (${{ matrix.image.image }})
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout bpfman
+        uses: actions/checkout@v4
 
-      - uses: actions/setup-go@v5
+      - name: Install Golang
+        uses: actions/setup-go@v5
         with:
           # prettier-ignore
           go-version: '1.22' # yamllint disable-line rule:quoted-strings
 
-      - uses: sigstore/cosign-installer@v3.5.0
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.5.0
 
       - name: Generate olm bundle on disk
-        if: ${{ matrix.image.image == 'bpfman-operator-bundle' }}
         run: |
           make bundle
```

Comment:
This needs to be `make bundle-build`. Since the manifests in https://github.com/bpfman/bpfman-operator/tree/main/bundle should always be up to date now

Reply:
Done